src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -436,10 +429,8 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None):
             if chunk_len < stride_left + stride_right:
                 raise ValueError("Chunk length must be superior to stride length")
 
-            rescale = self.type != "seq2seq_whisper"
-            # make sure that
             for item in chunk_iter(
-                inputs, self.feature_extractor, chunk_len, stride_left, stride_right, rescale, self.torch_dtype
```

Comment:
There is no need to rescale in `chunk_iter`.

1. For `type=seq2seq_whisper`, it was already doing nothing.
2. For `type=seq2seq`, stride is not used (see [here](https://github.com/huggingface/transformers/blob/80d50076c8d28a5f872066c825f9eb14eb737fbf/src/transformers/pipelines/automatic_speech_recognition.py#L552-L567))
3. For `type=ctc`, rescaling the stride is already done once [here](https://github.com/huggingface/transformers/blob/80d50076c8d28a5f872066c825f9eb14eb737fbf/src/transformers/pipelines/automatic_speech_recognition.py#L536-L539).

This wasn't spotted before because the rescaling ratio for ctc models was always 1 before, as pointed out in the previous comment.